### ISSUE TITLE
aria-controls='target'

### ISSUE THREAD

**Issue by oliverjam** [kentcdodds/react-toggled#9]
- `react-toggled` version: 1.1.2
- `node` version: 8.9.0
- `npm` (or `yarn`) version: 5.5.1

This is more of a question than a bug, so apologies if I've missed something obvious.

You set the `aria-controls` attribute to `'target'` in `getTogglerProps`, but I can't see this being set as an ID anywhere. Is 'target' a special value that `aria-controls` can take or is the ID just missing from the target element?

Thanks for this library by the way—it's a great intro to render props 👍

**Comment by kentcdodds** [comment]
You're welcome!

Actually, I'm not 100% certain about that attribute. I can't remember where I got the attribute. Is the value supposed to be an ID for another element?

**Comment by oliverjam** [comment]
Yup, it's supposed to reference the ID of the element it controls.

[According to Heydon](http://www.heydonworks.com/article/aria-controls-is-poop) it only works in JAWS and it doesn't exactly function as expected anyway. Sounds like the best solution is just to have the toggled region directly follow the toggle button (not sure how you enforce that in a library like this though)

**Comment by kentcdodds** [comment]
In that case, perhaps we should just remove it and folks can add it themselves easily enough. Would you like to do that? Feel free to [make a pull request](http://makeapullrequest.com) 👍

**Comment by oliverjam** [comment]
Yeah no problem, I can pull it out. Your `CONTRIBUTING.md` is 🔥 by the way

**Comment by kentcdodds** [comment]
Thanks!
